### PR TITLE
use tf.nn.batch_normalization if data_format == 'channels_last'

### PR DESCRIPTION
**Pull request by lgarithm**
### Checklist
- [ ] I've tested that my changes are compatible with the latest version of Tensorflow.
- [ ] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### Motivation and Context

in `BatchNormLayer` we re-implemented `tf.nn.batch_normalization` for additional support of `data_format == 'channels_first'`, this might caused performance issue for `data_format == 'channels_last'`.

### CODE REVIEW

**Comment by lgarithm** [comment]
@DEKHTIARJonathan could you test this PR to see if it is as fast as the original implementation for data_format == 'channels_last'?

I think reimplement `tf.nn.batch_normalization` to make it support `data_format == 'channels_first'` is still required, not just for the https://github.com/tensorlayer/openpose-plus project, but for all projects who uses `data_format == 'channels_first'` in BN. 

In fact this should be better fixed in TF not in TL, but it would take a much longer cycle to make the change in TF.

**Comment by lgarithm** [comment]
@luomai
